fuzzy.go

```diff
@@ -355,34 +358,21 @@ func (model *Model) EditsMulti(term string, depth int) []string {
 			break
 		}
 		for _, edit := range edits {
-			edits2 := Edits1(edit)
```

Comment:
to my surprise, this doesn't actually have any perf benefit, but I found it cleaner so I kept it.